lib/c-api/Cargo.toml

```diff
@@ -38,7 +38,7 @@ virtual-fs = { version = "0.11.4", path = "../virtual-fs", optional = true, defa
 enumset.workspace = true
 cfg-if = "1.0"
 lazy_static = "1.4"
-libc = { version = "^0.2", default-features = false }
+libc = { version = "=0.2.153", default-features = false }
```

Comment:
We really can't pin to a specific libc version, that will cause a lot of hassle for downstream users.

At most we can use a minimum version with the required features.

Reply:
This reminds me that we should use a workspace version for libc (only set in the root Cargo.toml)

Reply:
`libc 0.2.154` was yanked so cargo deny was failing.